src/components/events/CalendarEvents.jsx

```diff
@@ -28,11 +28,32 @@ const CalendarEvents = () => {
     },
   ];
 
+  const months = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+  ];
+
   return (
     <div className="relative flex flex-col">
       {/* Calendar Section */}
-      <section className="w-full flex-grow flex justify-center items-center">
-        <div className="w-full h-[95vh] relative">
+      <span
+        className="-rotate-90 text-biscuits-green-200 font-shrikhand text-3xl absolute md:translate-x-8 md:translate-y-[480px] -translate-x-12 translate-y-[440px]"
+        style={{ width: "120px", textAlign: "center" }}
+      >
+        {months[date.getMonth()]}
```

Comment:
https://stackoverflow.com/questions/1643320/get-month-name-from-date/18648314#18648314

try this instead